examples/MerminMagic/example.jl

```diff
@@ -0,0 +1,25 @@
+### A Pluto.jl notebook ###
```

Comment:
Since I was not using VSCode to develop, it's helpful to have a Pluto notebook to output some intermediate results during the development. I agree that it's not necessary to have a notebook.

To keep the format consistent, I will only keep the main file. I will remove it in the next PR.